functions/menu.bash

```diff
@@ -222,7 +226,7 @@ show_main_menu() {
     case "$choice2" in
       50\ *) backup_openhab_config ;;
       51\ *) restore_openhab_config ;;
-      *Restore\ text\ only*) restore_openhab_config "textonly" ;;
+      *Restore\ text\ only*) restore_openhab_config "${initialConfig:-/boot/initial.zip}" "textonly" ;;
```

Comment:
Can this file be changed by the user? Don't we want them to be able to choose a place to restore a text only config from if its not on the first boot/just after initial setup.

Reply:
Yes user can change the file but if the zipfile he wants to restore is elsewhere he can instead change the initialconfig setting in `openhabian.conf` to point there.
Plus if interactive this is just the proposed default the restore routine proposes when it makes a whiptail query.

Reply:
But if I read the logic correctly, for the textonly config we would never give the user a chance to select the file to restore from as a whiptail query or am I just reading the code wrong?

Reply:
So I hope but could be wrong as well.
textonly is now $2 ($1 is a new argument), in import_openhab_config() $1 is evaluated first (passed as proposal to whiptail)